sim/hunter/flanking_strike.go

```diff
@@ -65,6 +67,8 @@ func (hunter *Hunter) registerFlankingStrikeSpell() {
 		})
 	}
 
+	manaCostMultiplier := 1 - 0.02*float64(hunter.Talents.Efficiency)
```

Comment:
One way to handle this without updating every strike/sting etc would be to use an `OnSpellRegistered` callback and applying it to all relevant spells there based on a flag or specific spell codes. That might be a good refactor opportunity for later